cypress/e2e/zenscrape/login.cy.js

```diff
@@ -7,18 +7,17 @@ describe('Zenscrape Login page  ', () => {
 
   it('1. Test to click on the "Login" button and check the actual url', () => {
     cy.get('.bg-transparent').click();
-    cy.navigateUrlwithCookies(`${ZENSCRAPE_BASE_URL}login`);
+    cy.componentVisiblityCheck('.card-header','Login');
   });
-
+  
   it('2. Test to check the "Login" page should match the actual URL, if not redirect to the actual URL to ensure other test cases do not fail', () => {
     cy.url().then((currentUrl) => {
       if (!currentUrl.includes('/login')) {
         // Redirect to the actual URL if not already there
-        cy.visit('/login');
+        cy.visit('https://app.zenscrape.com/login');
```

Comment:
Why are we hardcoding the URL here?

Reply:
The base url((https://zenscrape.com/login)) is different and when we click on login button, the url is changing to https://app.zenscrape.com/login.

Reply:
@nidhiya-pious add this is the data and use a variable here.